### DISCUSSION TITLE
Welcome to deep-archive Discussions!

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/deep-archive#144] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nutjob4life** [comment]
Hi folks: Sean Kelly here, sometimes known by the unfortunate GitHub moniker as [nutjob4life](https://github.com/nutjob4life)—it's not an interesting story.

I helped develop the "Deep Archive" (which isn't to say I'm an expert at it) but I am familiar with its more intricate details so if you've got questions I can try to help!
